src/css/components/Panel.scss

```diff
@@ -73,6 +73,49 @@ $Panel-header-text-color: $cu-info !default;
   border-bottom-right-radius: $border-radius-large;
 }
 
+.Panel-body:last-child .Panel-row:last-child {
+  border-bottom-left-radius: $border-radius-large;
+  border-bottom-right-radius: $border-radius-large;
+}
+
+
+// Table Specific Stylings
+.Panel-body[role="table"] .Panel-row[role="row"] {
+  border-bottom: 0.5px solid $cu-divider;
```

Comment:
are these supposed to be color-neutral(5) like in above change? Just curious if we were moving away from using $cu-divider.